.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [10.12.0, 10.x, 12.x, 13.x, 14.x]
+        node-version: [10.12.0, 10.x, 12.x, 13.x, 14.x, 15.x, 16.x]
```

Comment:
I would remove 16.x from the test matrix for now as we know for certain the tests will not pass there yet.

Reply:
Yeah I adde there and wants to see if it can succeed. But I am not a member, so cannot see.
Dropped as requested.